pauthabielf64/pauthabielf64.rst

```diff
@@ -713,7 +958,7 @@ in the contents of the place. The relocation addend is used instead.
 ``ENCD(value, schema)`` is the encoding of the signing schema into the
 GOT slot.
 
-.. class:: pauthabielf64-signing-schema
+.. class:: pauthabielf64-signing-schema-alternative
```

Comment:
the class isn't actually used. since it looks like it has the same dimensios as the other table, perhaps the same class, `pauthabielf64-signing-schema`, will render it equally well?

Reply:
I can't find any documentation on what .. class:: actually does so given my limited understanding I'm going to remove them all for now.

Reply:
We use class to tell rst2pdf what the width of the columns should be. There's a style-sheet used when building: `tools/rst2pdf/rst2pdf-abi.style`. I think I should save us both some time and fix up the added tables myself in a follow-up commit.